### PR TITLE
[forge] add epoch change to forge (per-PR run)

### PR DESCRIPTION
**Pull request by igor-aptos**
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2541)

### CODE REVIEW

File: terraform/helm/genesis/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ chain:
   root_key: '0x5243ca72b0766d9e9cbf2debf6153443b01a1e0e6d086c7ea206eaf6f8043956'
   min_stake: 0
   max_stake: 100000
-  epoch_duration_secs: 86400 # 1 day
+  epoch_duration_secs: 120 # 2 minutes
```
Comment (sitalkedia) [inline terraform/helm/genesis/values.yaml:8]: I am not sure if we want to do it by default for every forge run - how do we think about having a nightly suite that can set very aggressive epoch change and run it for a longer duration to catch some of the regressions around epoch changes?

Comment (ikabiljo) [reply terraform/helm/genesis/values.yaml:8]: how do I change the config only for forge run for PRs? I think that is not possible yet, so using 2 minutes for all until then should be fine too?

I think we should have at least one or two epoch changes for push-blocking forge run (which lasts 5 minutes)?

Comment (sitalkedia) [reply terraform/helm/genesis/values.yaml:8]: @igor-aptos - How about we hold off on this while we add support for adding node configuration support for Forge? I will take a stab at it and hopefully we have it by early next week?

File: terraform/helm/aptos-node/values.yaml [hunk 2]
```diff
@@ -35,8 +35,8 @@ validator:
     provide_genesis: true
     sync_only: false
     concurrency_level: 8
-    round_initial_timeout_ms:
-    quorum_store_poll_count: 1
+    round_initial_timeout_ms: 1000
```
Comment (sitalkedia) [inline terraform/helm/aptos-node/values.yaml:38]: Can you clarify why is this needed? Its already set to 1000 as default right?

**Review by sitalkedia — APPROVED** [review]
As discussed offline, we can land this as a temp change. Once we support per run configuration set up, we can revert it from default config.

**Comment by sitalkedia** [comment]
@igor-aptos - Actually the latest run shows pretty significant regression - is that expected?

File: terraform/helm/aptos-node/values.yaml (hunk 2 above)
Comment (ikabiljo) [reply terraform/helm/aptos-node/values.yaml:38]: what does empty mean? should it be removed? I can leave it as is.

Comment (sitalkedia) [reply terraform/helm/aptos-node/values.yaml:38]: Yes, I think we can just remote it.
